react/week2/todo/src/components/Todo.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+
+function Todo(props) {
+  const [state, setDone] = useState([...props.items]);
+
+  const addTodo = () => {
+    // Add random item
+    const addRandomItem = (arr) => {
+      const newId = arr.length
+        ? Math.max(...arr.map((item) => item.id)) + 1
+        : 1;
+      const addRandomArr = arr.concat({
+        id: newId,
+        description: "Go to school",
+        done: false,
+      });
+      setDone(addRandomArr);
```

Comment:
If you take my advices from above this would be `setTodos(newTodos)`, which is more readable in my opinion